src/common/detector/Detector.h

```diff
@@ -166,4 +176,8 @@ class Detector {
   std::atomic_bool runThreads{true};
   uint32_t RuntimeStatusMask{0};
   bool CalibrationMode{false};
+
+  // Common to all EFUs
+  EV44Serializer *Serializer;
+  AR51Serializer *MonitorSerializer;
```

Comment:
I would put this as private and provide a const pointer to the users through a getter function, not a direct read/write access.

Reply:
Well,there are arguments for and against. I tend to be liberal on this matter ;-)